### ISSUE TITLE
Running the same command produces different number of dropped objects

### ISSUE THREAD

**Issue by ismael-mendoza** [DarkEnergyScienceCollaboration/WeakLensingDeblending#45]
There is an issue with the current version of WLD. I realized that when running the _exactly same_ command multiple times, like: 
```
./WeakLensingDeblending/simulate.py --catalog-name params/OneDegSq.fits --image-width 1800 --image-height 1800 --ra-center -0.05 --dec-center -0.45 --survey-name LSST --filter-band i --output-name demo1--verbose --no-stamps --no-hsm --no-agn
```
will produce different number of dropped objects from the fisher inversion process. To be precise running: 
```
sum(cat[cat['snr_grpf']==0]) 
```
may produce different results even even if `cat` was created from the exact same command.

**Comment by ismael-mendoza** [comment]
Should be fixed after #44 is merged. The version after the PR consistently results in 
```
sum(cat[cat['snr_grpf']==0]) = 28 
```
for multiple runs of the command above.

**Comment by ismael-mendoza** [comment]
Resolved with #44  (although exact reason for this error remains unknown)
